doc/cephadm/monitoring.rst

```diff
@@ -0,0 +1,70 @@
+Monitoring Stack with Cephadm
+=============================
+
+The Ceph dashboard makes use of prometheus, grafana, and related tools
+to store and visualize detailed metrics on cluster utilization and
+performance.  Ceph users have three options:
+
+#. Let cephadm deploy and configure these services in an opinionated way.
```

Comment:
```suggestion
#. Enabling cephadm to deploy and configure these services in an opinionated way.

Reply:
I suggest re-wording "opinionated way" as this applies a human interaction to a service. Unless you're trying to say that the user can utilise cephadm in an opinionated way, but even then I'm not sure what that means in this context.